Healthy/Classes/Modules/Dashboard/Views/HomeHeaderView.xib

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
<img width="299" alt="Screenshot 2023-06-14 at 1 46 33 AM" src="https://github.com/motoon-eg/healthy/assets/45182214/2f76b854-e88d-4289-909a-277e8ecd1c62">

Top space is not correct. This should be controlled from the parent and will be inferred later using the safe area.